lighthouse-core/scripts/legacy-javascript/update-sizes.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+for d in variants/* ; do
+  echo "$d"
+  wc -c "$d"/*/main.bundle.min.js | sort -hr
+  echo -e "\n"
+done > summary-sizes.txt
```

Comment:
bash is wonderful sometimes

Reply:
getting this into the readme mix would be great too :D